foundations/installations/prerequisites.md

```diff
@@ -254,6 +254,11 @@ Currently there is a bug preventing CloudReady v83.4 from successfully installin
 
 ### Google Chrome Installation
 
+#### Why Google Chrome?
+
+Because our lessons use Google Chrome, and Chrome/Chromium are overwhelmingly used by devs and consumers, The recommendations we make are very intentional.
+[Look at this browser market and see what other folks are using the most](https://gs.statcounter.com/browser-market-share#monthly-202110-202110-bar)
```

Comment:
```suggestion
Look at this [breakdown of browser market shares](https://gs.statcounter.com/browser-market-share#monthly-202110-202110-bar) and see what other folks are using the most
```

Reply:
Thanks for the better wording <3

Reply:
You're welcome! We're recently striving to make link text more accessible.